pkg/scheduler/framework/types.go

```diff
@@ -776,6 +773,13 @@ func (n *NodeInfo) resetSlicesIfEmpty() {
 	}
 }
 
+func max(a, b int64) int64 {
```

Comment:
We could use `math.Max` (though we would need to cast to float64 then back), so that's not totally a blocker on this for me

Reply:
Thanks a lot for your review and suggestion!
My personal habit is to write a `max` or `min` function directly for simple types ([aha, I was influenced by this article ^-^](https://mrekucci.blogspot.com/2015/07/dont-abuse-mathmax-mathmin.html)).